classes/core/PKPApiRoutingHandler.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace PKP\core;
+
+use PKP\handler\APIHandler;
+
+class PKPApiRoutingHandler extends APIHandler
```

Comment:
Odd -- I can't see where this class is attached to anything! What am I missing?

Reply:
This is to facilitate our current implementation of API routing system . Our current `APIRouter` required a handler that extends `APIHandler` to work . But if we plan to use laravel routing system , we need controller . So this one single class will work as a bridge between those 2 . See how it's use for [User](https://github.com/pkp/ojs/pull/3980/files#diff-a67dc7f90e012f5dfa7eef2a5424a828c1d83a6ef1017d8509615d414de18fa9) and [Job](https://github.com/pkp/ojs/pull/3980/files#diff-cdfbe01ee2eb75a66b1d59b1af4625d2e48331be728e64ad7973c08da4ea9f7c) how it's used where we inject the controller class itself .